datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_traps_db.py

```diff
@@ -213,12 +225,12 @@ def write_compact_trap_db(trap_db_per_mib, output_file, use_json=False):
         for trap_oid, trap in trap_db["traps"].items():
             if trap_oid in compact_db["traps"] and trap["name"] != compact_db["traps"][trap_oid]["name"]:
                 echo_warning(
-                    "Found name conflict for trap OID {}, ({} != {}). Will ignore".format(
-                        trap_oid, trap['name'], compact_db["traps"][trap_oid]["name"]
+                    "Found name conflict for trap OID {}, ({}::{} != {}::{}). Will ignore".format(
+                        trap_oid, mib, trap['name'], compact_db["traps"]['mib'], compact_db["traps"][trap_oid]["name"]
                     )
                 )
                 conflict_oids.add(trap_oid)
-            compact_db["traps"][trap_oid] = trap
+            compact_db["traps"][trap_oid] = {"mib": mib} | trap
```

Comment:
Is `|` a valid operand in example above ?

I'm getting:

```
>>> a = {"a":"b"} | {"c": 2}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for |: 'dict' and 'dict
```

Reply:
It's python3 only syntax, which is what ddev is intended for now

Reply:
We don't support > 3.8 https://www.python.org/dev/peps/pep-0584/

Reply:
Oh good call, thanks! Updating